yt_dlp/extractor/piapro.py

```diff
@@ -88,25 +88,19 @@ def _real_extract(self, url):
         if category_id not in ('1', '2', '21', '22', '23', '24', '25'):
             raise ExtractorError('The URL does not contain audio.', expected=True)
 
-        str_duration, str_filesize = self._search_regex(
-            r'サイズ：</span>(.+?)/\(([0-9,]+?[KMG]?B)）', webpage, 'duration and size',
-            group=(1, 2), default=(None, None))
-        str_viewcount = self._search_regex(r'閲覧数：</span>([0-9,]+)\s+', webpage, 'view count', fatal=False)
+        str_duration = self._search_regex(r'長さ：\s*(.+?)</p>', webpage, 'duration')
+        str_filesize = self._search_regex(r'サイズ：\s*([0-9.]+?[KMG]?B)', webpage, 'size')
+        str_viewcount = self._search_regex(r'閲覧数：\s*([0-9,]+)\s*</p>', webpage, 'view count', fatal=False)
```

Comment:
I would slightly adjust the regex and make them all non-fatal
```suggestion
        str_duration = self._search_regex(r'長さ：\s*([\d:]+)\s*</p>', webpage, 'duration', fatal=False)
        str_filesize = self._search_regex(r'サイズ：\s*([\d.]+[KMG]?B)\s*</p>', webpage, 'size', fatal=False)
        str_viewcount = self._search_regex(r'閲覧数：\s*([\d,]+)\s*</p>', webpage, 'view count', fatal=False)
        create_date = self._search_regex(r'投稿日：\s*([\d\s/:]+)\s*</p>', webpage, 'timestamp', fatal=False)
```